apache2.yaml

```diff
@@ -142,9 +148,7 @@ subpackages:
     pipeline:
       - runs: |
           set -x
-
-          # Install the entrypoint script
-          install -Dm755 entrypoint.sh "${{targets.subpkgdir}}"/usr/bin/entrypoint.sh
+          export short_version=$(echo ${{package.version}} | cut -d. -f1-2)
 
```

Comment:
is this intentionally dropping entrypoint.sh? is it no longer needed?
do we need short_version? seems unused.

Reply:
Yea, unintentional. Removed